src/events.cpp

```diff
@@ -192,7 +192,22 @@ bool Events::eventCreatureOnChangeOutfit(Creature* creature, const Outfit_t& out
 
 	LuaScriptInterface::pushOutfit(L, outfit);
 
-	return scriptInterface.callFunction(2);
+	bool success = true;
+
+	if (scriptInterface.protectedCall(L, 2, 1) != 0) {
+		success = false;
+		LuaScriptInterface::reportError(nullptr, LuaScriptInterface::popString(L));
+	} else {
+		if (LuaScriptInterface::isTable(L, -1)) {
+			outfit = LuaScriptInterface::getOutfit(L, -1);
```

Comment:
getOutfit does not work with relative indices, I suppose that 1 is fine, since the return value will always be in this position, unless there is something else that I do not know?